.github/ISSUE_TEMPLATE/BUG-REPORT.yml

```
@@ -39,7 +63,21 @@ body:
     id: logs
     attributes:
       label: Relevant log output
-      description: Please copy and paste any relevant log output. This will be automatically formatted into code, so no need for backticks.
+      description: |
+        Please paste relevant logs that were created when reproducing the error.
+        
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)